### ISSUE TITLE
Spreadsheet import error

### ISSUE THREAD

**Issue by maccraymer** [ISO-TC211/iso-geodetic-registry#77]
The spreadsheet import function gives a "504 Gateway Time-out" error when trying to load a single new transformation into the Registry. Because the spreadsheet importing worked in the software version running in Norway, it appears the error is related to it's installation at Ribose. This is a critical function as it is presently the only way we are able to enter multiple items into the Registry.

**Comment by ronaldtse** [comment]
Got it. @maccraymer would you mind sharing the file so we can try reproducing it? Thanks.

**Comment by maccraymer** [comment]
Here's an example. I went to the Coordinate Operation > Transformations part of the GUI and clicked the Upload Proposal button above the search box.
[Geodetic_Register_CONTENT_TEMPLATE - Canada 20200113.xlsx](https://github.com/ISO-TC211/iso-geodetic-registry/files/4404525/Geodetic_Register_CONTENT_TEMPLATE.-.Canada.20200113.xlsx)

**Comment by ronaldtse** [comment]
@phuonghuynh can you help check this out? Thanks!

**Comment by phuonghuynh** [comment]
@ronaldtse is there any document related to excel parser logic? the code failed at this row, but I not sure what should I fix it. The reason is that we don't have `itemIdentifier=95`

![Screenshot_3](https://user-images.githubusercontent.com/705510/79035684-223a5280-7beb-11ea-993d-5f0a905f710b.png)

**Comment by maccraymer** [comment]
@phuonghuynh the item identifier is a reference to items in the existing Registry. It can be found in the public Registry at

[https://geodetic.isotc211.org/item/9cb38505-edbf-4cf3-a465-e067d3d96ee9](url)

and it can be found in the staging Registry at

[https://geodetic-staging.isotc211.org/item/9cb38505-edbf-4cf3-a465-e067d3d96ee9](url)

**Comment by florianesser** [comment]
@phuonghuynh I don't think that this is caused by the Excel parser. I imported Mike's example successfully in my development setup by adapting the item identifier references (e.g. `95 - NTv2`) to the respective IDs in my database.

Could this be a configuration problem in the AWS setup that prevents/mishandles the `POST` request to `/register/{uuid}/proposal/upload` and leads to the `504 (Gateway timeout)`?

**Comment by phuonghuynh** [comment]
Production site now should have this fix.

**Comment by maccraymer** [comment]
I get a new HTTP Status 500 error when importing an Excel spreadsheet for Korea (attached).

[Geodetic_Register_CONTENT_TEMPLATE_v2_Korea_20200518.xlsx](https://github.com/ISO-TC211/iso-geodetic-registry/files/4643006/Geodetic_Register_CONTENT_TEMPLATE_v2_Korea_20200518.xlsx)

However, I am able to import a Chinese spreadsheet with similar information (attached).

[Geodetic_Register_CONTENT_TEMPLATE_v2_China_20200515.xlsx](https://github.com/ISO-TC211/iso-geodetic-registry/files/4642992/Geodetic_Register_CONTENT_TEMPLATE_v2_China_20200515.xlsx)

**Comment by ronaldtse** [comment]
@phuonghuynh could you help have a look at what error this is? Thanks!

**Comment by phuonghuynh** [comment]
Yes, checking

**Comment by maccraymer** [comment]
Doh! I just realized what this error is coming from. The spreadsheet was an old one (just copied the contents into the new spreadsheet template). I see now that it contains references to items in the master spreadsheet, not the actual Geodetic Registry that has since been put online! I will correct all the references to existing items to use the actual Registry item numbers (not those form the master spreadsheet). Sorry for wasting your time!!

**Comment by maccraymer** [comment]
Well...another doh moment that contradicts the previous doh moment! I looked further and see that the Korean spreadsheet IS using references to items numbers in the current online Registry. I will have to dig deeper into why this spreadsheet is failing to import unless @phuonghuynh can find it first ;-)
